MediaGallerySynchronization/etc/queue_topology.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework-message-queue:etc/topology.xsd">
+    <exchange name="magento-db" type="topic" connection="db">
+        <binding id="MediaGalleryIndexer" topic="media.gallery.indexer"
```

Comment:
```suggestion
        <binding id="media.gallery.synchronization" topic="media.gallery.synchronization"
```
Please rename all `indexer` occurrences to `synchronization` to avoid the confusion with Magento indexers